modules/cactus-web/src/Divider/Divider.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components'
+
+import { border } from '../helpers/theme'
+
+export const Divider = styled.hr`
+  border: ${(p) => border(p.theme, 'lightContrast')};
```

Comment:
This line looked thicker than 1px to me, so I did some digging and found this: https://stackoverflow.com/questions/4151743/how-can-i-change-the-thickness-of-my-hr-tag

Basically it recommends changing this to the following.

```suggestion
  border: none;
  height: ${(p) => p.border === 'thin' ? '1px' : '2px'};
  color: ${(p) => p.theme.colors.lightContrast};
  background-color: ${(p) => p.theme.colors.lightContrast};
```

The attached screenshot shows both the new styles (top divider) and the styles as you have them (bottom divider).  Notice how the new styles are thinner.

<img width="298" alt="Screen Shot 2021-03-02 at 3 28 16 PM" src="https://user-images.githubusercontent.com/46460514/109711105-7c7a3400-7b6c-11eb-8119-309d7ce56315.png">